.github/workflows/release.yml

```
@@ -89,26 +88,51 @@ jobs:
         with:
           ref: main
 
+      - name: Configure git
+        run: |
+          git config --global user.name "edgelessci"
+          git config --global user.email "edgelessci@users.noreply.github.com"
+
       - name: Create docs release
+        if: inputs.kind == 'minor'
         working-directory: docs
         run: |
           npm ci
           npm run docusaurus docs:version "${MAJOR_MINOR}"
+          git add .
+          git commit -am "docs: release ${MAJOR_MINOR}"
+          # Clean up auxiliary files, so next steps run on a clean tree
+          git clean -fdx :/
+
+      - name: Update version.txt
+        if: inputs.kind == 'minor'
+        run: |
+          pre_release_version="v${{ needs.verify-inputs.outputs.PART_MAJOR }}.$((${{ needs.verify-inputs.outputs.PART_MINOR }} + 1)).${{ needs.verify-inputs.outputs.PART_PATCH }}-pre"
+          echo "${pre_release_version}" > version.txt
+          git add version.txt
+          git commit -am "chore: update version.txt to ${pre_release_version}"
```

```suggestion
          git add version.txt
          git commit -m "chore: update version.txt to ${pre_release_version}"
```

This makes sure that no leftovers are accidentally added. Alternatively, only do `-a` without `add`.